templates/4a_compensate_image_data_round2.ipynb

```diff
@@ -0,0 +1,416 @@
+{
```

Comment:
<p>Given that this matrix contains all zeros, there isn't a scenario where it would be used for rosetta.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Would it be better to provide a default matrix that does have some channel-channel values set, and tell the user to replace the default matrix with their own?

Reply:
Given that this is going to be project specific, we won't know what the defaults are. I was saying that the text needs to be updated to reflect this fact, not that we will need to change the default.

Reply:
Got it.

Reply:
<p>I am guessing the user will edit the "commercial_rosetta_matrix_round2.csv" which is what default_matrix_path is pointing to? </p><p> </p><p>If thats the case maybe alter this text to reflect that - "You will need to provide your own&nbsp;<code>.csv</code>&nbsp;which contains the values you need." </p>